questionable.gemspec

```diff
@@ -7,8 +7,8 @@ require "questionable/version"
 Gem::Specification.new do |s|
   s.name        = "questionable_surveys"
   s.version     = Questionable::VERSION
-  s.authors     = ["Nick Urban, Dorian Marié"]
-  s.email       = ["nick@nickurban.com, dorian@bespokepost.com"]
+  s.authors     = ["Nick Urban, Dorian Marie, Jared Hales"]
```

Comment:
https://github.com/bespokepost/questionable/issues/9 - I don't know why that character would be an issue. Maybe something in his stack doesn't support unicode characters?